include/vstd/StringUtils.cpp

```diff
@@ -0,0 +1,12 @@
+#include "StringUtils.h"
+
+namespace vstd {
```

Comment:
put `{` on new line and do not indent namespace contents. We indent namespace if and only if it contains forward declarations

Reply:
I don't really like such inconsistency. Will make work on automated formatting much harder...

Reply:
Agreed regarding inconsistency. Also, whatever we decide should probably go into https://wiki.vcmi.eu/Coding_guideline.